public/public_libraries/strype/graphics.py

```diff
@@ -849,3 +849,30 @@ def stop():
     Stops the whole execution immediately.  Will not return.
     """
     raise SystemExit()
+
+_last_frame = _time.time()
+
+def pause(actions_per_second):
+    """
+    Waits for a suitable amount of time since the last call to pause() to allow you to take
+    the given number of actions per second.  This is almost always used like follows:
+    
```

Comment:
I have some difficulty to understand the part "the given number of actions per second" in the comment. Later you say "Where 30 is the number of times you want to do those actions per second.", which I think is clear, so did you mean to say "the given number of times per second" in the comment here?

Reply:
Ok, I've adjusted it (and also added a default value for the parameter, so many users may not need to think about it).